src/main/java/make_post/interface_adapters/MakePostController.java

```diff
@@ -20,23 +20,29 @@ public MakePostController(MakePostInputBoundary interactor){
         this.interactor = interactor;
     }
 
+    /**
+     * passes the input made by the user to make the post to the interactor.
+     * @param postBody the attributes of the post input by the user.
+     */
     public void passToMakePostInteractor(Map<String, Object> postBody){
```

Comment:
Nitpick but you could just name this something like `execute` or `makePost` or something. As it is, it's kinda too technical.

Reply:
What do you mean by the comment about the path containing part of the directory?